pkg/storage/db_test.go

```diff
@@ -199,6 +199,72 @@ func TestDBEmptyNamespace(t *testing.T) {
 	}
 }
 
+func TestDBExistsNoNamespaceKey(t *testing.T) {
+	for _, dbImpl := range getDBImplementations(t) {
+		db := dbImpl
+
+		exists, err := db.Exists(context.Background(), "dnenamespace", "dnekey")
+		assert.NoError(t, err)
+		assert.False(t, exists)
+	}
+}
```

Comment:
This is awesome! Small suggestion to also include the expected behavior in the test, so reader can quickly verify that what you're asserting it correct.
```go
func TestDBExists_FalseWhenNoNamespaceKey(t *testing.T) {
	for _, dbImpl := range getDBImplementations(t) {
		db := dbImpl

		exists, err := db.Exists(context.Background(), "dnenamespace", "dnekey")
		assert.NoError(t, err)
		assert.False(t, exists)
	}
}
```